### PR TITLE
Ci build script

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: .github/workflows/bob.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.2'
+
+      - name: Get Defold version
+        run: |
+          TMPVAR=`curl -s http://d.defold.com/stable/${{env.VERSION_FILENAME}} | jq -r '.sha1'`
```
Comment (JCash) [inline .github/workflows/bob.yml:62]: "jq" saves us from the trouble of using additional external tools to grab a json property

File: .github/workflows/bob.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          echo "DEFOLD_VERSION=${TMPVAR}" >> $GITHUB_ENV
```
Comment (JCash) [inline .github/workflows/bob.yml:63]: By writing to GITHUB_ENV, the next action will get an updated environment.

File: .github/workflows/trigger-site-rebuild.yml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 name: Trigger site rebuild
 
-on: [push]
+on:
+  push:
+     branches:
+       - 'master'
```
Comment (JCash) [inline .github/workflows/trigger-site-rebuild.yml:6]: If felt wrong that this was triggered for each commit I made on a dev branch

File: .github/workflows/bob.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+          echo "Found version ${TMPVAR}"
+
+      - name: Download bob.jar
+        run: |
+          wget -q http://d.defold.com/archive/stable/${{env.DEFOLD_VERSION}}/bob/bob.jar
+          java -jar bob.jar --version
+
+      - name: Resolve libraries
+        run: java -jar bob.jar resolve --email a@b.com --auth 123456
```
Comment (britzl) [inline .github/workflows/bob.yml]: I believe we recently removed the need for specifying a fake email and auth on public repos

**Review by britzl — APPROVED** [review]
Looks great! A note on --email and --auth.

**Pull request merged**